src/cmdc_tools/datasets/official/DE/data.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+import requests
+import us
+
+from ...base import DatasetBaseNoDate
+from ..base import CountyData
+
+
+class Delaware(DatasetBaseNoDate, CountyData):
+    state_fips = int(us.states.lookup("Delaware").fips)
+    has_fips: bool = False
+    source_map = {
+        "Sussex": "https://myhealthycommunity.dhss.delaware.gov/locations/county-sussex/download_covid_19_data",
+        "Kent": "https://myhealthycommunity.dhss.delaware.gov/locations/county-kent/download_covid_19_data",
+        "New Castle": "https://myhealthycommunity.dhss.delaware.gov/locations/county-new-castle/download_covid_19_data",
+    }
+    source = list(source_map.values())
```

Comment:
It will handle things by the primary key. 

For the data and logging tables the pk is (vintage, dt, location, variable_id)

We don't yet have a way to inject a dict or list of sources, but the db is ready for it